new_client/src/components/blogs/Blog.jsx

```diff
@@ -170,13 +184,36 @@ const Blogs = (props) => {
                       Estimated reading time - {" "}
                       {calculateReadingTime(article.blogContent)}
                     </p>
+                    <span>Tags - </span>
+                    <ul 
+                    style={{ 
+                      display: "inline-flex", 
+                      listStyleType:"none",
+                    }}>
+                      {article.tags.length !== 0 ? article.tags.map(t => (
+                        <li
+                          key={t}
+                          style={{
+                            padding: `2px 6px`,
+                            marginRight: `5px`,
+                            cursor: `pointer`,
+                          }}
+                          onClick={e =>
+                            props.history.push(
+                              `/blogs?tag=${t}`
+                            )
+                          }>
+                          {t}
+                        </li>
+                      )) : <li>No Tags</li>}
```

Comment:
We should not explicitly mention 'No Tags', we can render this component conditionally only.

Reply:
When the component was not being rendered the card lengths were misaligned. But okay I'll change this thing.